### ISSUE TITLE
ios-clang aborts

### ISSUE THREAD

**Issue by radare** [waneck/linux-ios-toolchain#8]
Can't even show the help
```
$ ios-clang
terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::substr: __pos (which is 18446744073709551615) > this->size() (which is 0)
Aborted
```

**Comment by radare** [comment]
`ldid` is the only installed program that works.

**Comment by PDeveloper** [comment]
+1 It would be great to get info on this bug.

**Comment by PDeveloper** [comment]
Seconds after posting this, and second google search result:
http://www.ninthavenue.com.au/how-to-build-an-ios-toolchain-for-linux-debian-7

**Comment by tpoechtrager** [comment]
@waneck: Maybe you could switch to my [wrapper](https://github.com/tpoechtrager/cctools-port/blob/master/usage_examples/ios_toolchain/wrapper.c)? My wrapper shouldn't throw any exceptions :smiley:.

**Comment by waneck** [comment]
@tpoechtrager yeah, I think I'll just switch the projects entirely :) Your fork of cctools seems more up-to-date also
